decidim-elections/app/assets/javascripts/decidim/elections/onboarding.js.es6

```diff
@@ -0,0 +1,5 @@
+
+$(async () => {
```

Comment:
The `async` keyword is not necessary. Remember: no `await`, no `async` 😄

Reply:
🤦‍♀️ Gosh, yeah.